### ISSUE TITLE
Can't install vagrant (RPM/gem) on Redhat/CentOS 5

### ISSUE THREAD

**Issue by codebling** [hashicorp/vagrant#958]
tl;dr
vagrant cannot be installed by RPM or gem on CentOS 5. RPM was broken because of an FPM bug which has been closed and the fix released in a new build. 

Long version:
As mentioned on Google Groups thread https://groups.google.com/forum/#!msg/vagrant-up/rP9zeqq6f9o/r-QIBayHTiwJ

It's mentioned that the issue is in FPM:
https://github.com/jordansissel/fpm/issues/192

The FPM issue is now closed and the fix released in a new build (https://github.com/jordansissel/fpm/zipball/v0.4.10). 

It would be cool to get some new RPM drops because the suggested workaround - that vagrant can be installed via gem - doesn't work. The gem install fails with a buffer overflow. This is supposedly because, from what I've read, CentOS 5 is still using ruby 1.8.5. It's supposed to work with later versions of ruby. 

Here's what installing from source looks like:
So ok, let's install ruby from source.. yum remove ruby, wget ruby, ./configure, make, make install. Ok, done! Wow, this is easy. Let's install gems now. Wait, it's complaining about yaml. Damnit. Ok, yum install libyaml-dev, rebuild ruby. Let's try again. No wait, that didn't work, yum remove yamldev, wget yaml, ./configure, make, make install, rebuild ruby. Ok, now install gems. Wait, it's complaining about zlib. Damnit. yum install zlib, rebuild ruby, install gems. No dice. Configure the ruby extension, rebuild ruby, install gems. Still no good. <give up> 

Basically, it's one hell of a workaround and I'm not even sure it can work. Other than that, there's no way to install vagrant on CentOS 5, which is kind of unfortunate because its end of life is still 5 years away. This was on 5.3. It might work better on later versions, but I'm not sure about that.
